specification/communication/data-plane/PhoneNumbers/preview/2021-10-25-preview/phonenumbers.json

```diff
@@ -0,0 +1,1639 @@
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Release phone number": {
+            "$ref": "./examples/ReleasePhoneNumber.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ]
+      }
+    },
+    "/phoneNumbers": {
+      "get": {
+        "tags": [
+          "Manage purchased phone numbers"
+        ],
+        "summary": "Gets the list of all purchased phone numbers.",
+        "operationId": "PhoneNumbers_ListPhoneNumbers",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "An optional parameter for how many entries to skip, for pagination purposes. The default value is 0.",
+            "type": "integer",
+            "format": "int32",
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "top",
+            "description": "An optional parameter for how many entries to return, for pagination purposes. The default value is 100.",
+            "type": "integer",
+            "format": "int32",
+            "default": 100
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PurchasedPhoneNumbers"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "phoneNumbers"
+        },
+        "x-ms-examples": {
+          "List purchased phone numbers": {
+            "$ref": "./examples/GetPhoneNumbers.json"
+          }
+        }
+      }
+    },
+    "/shortCodes": {
+      "get": {
+        "tags": [
+          "ShortCodes"
+        ],
+        "summary": "Gets the list of short codes for the current resource.",
+        "operationId": "ShortCodes_GetShortCodes",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "An optional parameter for how many entries to skip, for pagination purposes.",
+            "type": "integer",
+            "format": "int32",
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "take",
```

Comment:
`take` isn't one of the blessed terms from https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options.  I think your use here corresponds to `top`.

Reply:
You are absolutely right. Thanks for pointing this out. It turns out that there are other endpoints in the Telephony Service, that are not part of this Swagger, that are using the name "top". Luckily, the existing endpoint on this Swagger that was already GAd is using "take" so I'll rename these.